### PR TITLE
Add payment captured-by meta data

### PR DESCRIPTION
**Pull request by dkubb**

### CODE REVIEW

File: backend/app/controllers/spree/admin/payments_controller.rb [hunk 1]
```diff
@@ -1,51 +1,45 @@
...
         else
-          @payment_method = @payment_methods.first
+          payment.public_send("#{event}!")
         end
       end
 
-      def load_order
-        @order = Order.find_by_number!(params[:order_id])
+      def order
+        @order ||= Order.find_by_number!(params.fetch(:order_id))
```
Comment (dkubb) [inline backend/app/controllers/spree/admin/payments_controller.rb]: I wonder if I should refactor this to `Order.find_by!(number: params.fetch(:order_id))`

Comment (mbj) [inline backend/app/controllers/spree/admin/payments_controller.rb]: Yeah, I was about to flag that. We normally do not use the dynamically created methods.

Comment (dkubb) [inline backend/app/controllers/spree/admin/payments_controller.rb]: Fixed.

File: core/db/structure.sql [hunk 2]
```diff
@@ -730,6 +730,7 @@ CREATE TABLE spree_payment_capture_events (
     payment_id identifier,
     created_at create_timestamp,
     updated_at update_timestamp,
+    created_by_id identifier,
```
Comment (mbj) [inline core/db/structure.sql]: I wounder if this one can be set `NOT NULL` ?

Comment (dkubb) [inline core/db/structure.sql]: There's over `776700` records in that table already, and I'd prefer not to remove that data.

Comment (mbj) [inline core/db/structure.sql]: Can we backfill these with a reasonable value? Like making them captured by stephanie?

Comment (mbj) [inline core/db/structure.sql]: On the spree side this is not the case. Lets make it `NOT NULL` and deal with the difference in the main app.

Comment (dkubb) [inline core/db/structure.sql]: What if we set it equal to whomever captured the order? That won't fill it in 100% of the time, but it's better than setting *everything* to stephanie first.

Comment (mbj) [inline core/db/structure.sql]: Much better.

**Pull request merged**